Surface_mesh_segmentation/include/CGAL/boost/graph/alpha_expansion_graphcut_maxflow.h

```diff
@@ -0,0 +1,103 @@
+#ifndef CGAL_BOOST_GRAPH_ALPHA_EXPANSION_GRAPHCUT_MAXFLOW_H
+// Copyright (c) 2014  GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0+
```

Comment:
License issue: the CGAL package `BGL` is supposed to be LGPLv3+.

Reported by Travis: https://travis-ci.org/CGAL/cgal/jobs/562464389#L2760

Reply:
This file is in Surface Mesh Segmentation for this very reason (see the __License__ section of my small feature), I don't understand the Travis error.